### ISSUE TITLE
ical export is not RFC 5545 compliant (DTSTART field)

### ISSUE THREAD

**Issue by hiqua** [lfos/calcurse#81]
ICSDroid does not parse the ical export from calcurse correctly, so I reported it on the ICSDroid side, but Ricki Hirner from ICSDroid pointed me to the following which indicates that the bug is actually in calcurse.

According to RFC 5545 (https://tools.ietf.org/html/rfc5545#section-3.8.2.4)
```
3.8.2.4.  Date-Time Start

   Property Name:  DTSTART

   Purpose:  This property specifies when the calendar component begins.

   Value Type:  The default value type is DATE-TIME.  The time value
      MUST be one of the forms defined for the DATE-TIME value type.
      The value type can be set to a DATE value type.
```
and (https://tools.ietf.org/html/rfc5545#section-3.3)
```
3.3.  Property Value Data Types

   The properties in an iCalendar object are strongly typed.  The
   definition of each property restricts the value to be one of the
   value data types, or simply value types, defined in this section.
   The value type for a property will either be specified implicitly as
   the default value type or will be explicitly specified with the
   "VALUE" parameter.  If the value type of a property is one of the
   alternate valid types, then it MUST be explicitly specified with the
   "VALUE" parameter.
```
which means that whenever a DATE field (instead of DATE-TIME) has to be specified for DTSART (or DTEND), it has to be specified with the VALUE tag, such as:
```
       DTSTART;VALUE=DATE:20070628
       DTEND;VALUE=DATE:20070709
```
instead of:
```
       DTSTART:20070628
       DTEND:20070709
```
as is currently the case. These examples are from the RFC 5545 as well.

Although calcurse mentions RFC 2445, it is superseded by RFC 5545, so I think this is relevant.

Once the issue is acknowledged, I can create a PR.

**Comment by hiqua** [comment]
I guess this means that you agree about the issue, so yeah I'll try to work on this, it looks easy enough for me.

**Comment by lfos** [comment]
Fixed by 0114289 (ical: use the VALUE parameter in DTSTART, 2018-05-09).
